app/assets/stylesheets/application.scss

```diff
@@ -1,5 +1,8 @@
 // shared styles
 @import "shared/shared";
 
+// app styles
+@import "app";
```

Comment:
I was under the impression that the `shared` directory is for styles shared between apps. I added the `app` directory to organize app and page specific styles. Wanted to avoid adding sass files in the root of `stylesheets` since those will get processed with each page build. Files in a subdirectory don't, unless they are imported.

Reply:
Yeah, `/shared` will (hopefully, someday) be a git submodule.  With shared SCSS between all apps.

Do all top-level files get processed?  I was under the impression that rails loads the entrypoint only (`application.scss`).

I'm fine with whatever scheme, as long as we're consistent.  But I threw that `fake.scss` in the top level, so we may have to reorganize that.

Reply:
Yep. All `.scss` files get processed even though they are not loaded anywhere. You can test this by creating a `foo.scss` file and adding `.foo { color: $blue; }` as a style in it. Don't add any imports for it. Restart the rails server so the sass processor will pick it up. You will find that any page load after that will error on that undefined `$blue` variable.

Reply:
I pushed up a change that renames the `audio-segmenter` directory and removes the fake page stylesheet (it was empty aside from a redundant import of bootstrap variables.)